docs/source/install/ewc_ha.rst

```diff
@@ -0,0 +1,287 @@
+|bwc| HA Cluster - BETA
+=======================
+
+|bwc| (EWC) is the commercial version of the StackStorm automation platform. EWC adds priority
+support, advanced features such as fine-tuned access control, LDAP, and Workflow Designer. To
+learn more about |bwc|, get an evaluation license, or request a quote, visit `extremenetworks.com/product/workflow-composer
+<https://www.extremenetworks.com/product/workflow-composer/>`_.
+
+This document provides an installation blueprint for a Highly Availabile StackStorm Enterprise (|bwc|) cluster
+based on `Kubernetes <https://kubernetes.io/>`__, a container orchestration platform at planet scale.
+
+The cluster deploys minimum of 2 replicas for each component of StackStorm microservices for redundancy and reliability,
+as well as configures backends like MongoDB HA Replicaset, RabbitMQ HA and etcd cluster that st2 relies on for database,
+communication bus, and distributed coordination respectively. That raises a fleet of more than ``30`` pods total.
+
+The source code for K8s resource templates is available as a GitHub repo (TODO):
+`StackStorm/stackstorm-enterprise-ha <https://github.com/StackStorm/stackstorm-enterprise-ha>`_.
```

Comment:
### TODO

- [ ] Merge https://github.com/StackStorm/k8s-st2/pull/78
- [ ] Extract Helm chart to https://github.com/StackStorm/stackstorm-enterprise-ha
- [ ]  Open https://github.com/StackStorm/stackstorm-enterprise-ha repo
- [ ] Copy issues from `k8s-st2` to `stackstorm-enterprise-ha` repo